### ISSUE TITLE
The Spleen dataset，How to handle it for this dataset，Prepare_your_dataset.py，This one can't deal with the spleen_37.nii.gz

### ISSUE THREAD

**Issue by Alex-lcy** [wurenkai/H-vmunet#11]
The Spleen dataset was released using Antonelli et al. [2] to release the Spleen dataset, sourced from
Memorial Sloan Kettering Cancer Center, which includes, 41 cases for the training set and 20 cases
for the test set. Since the labels of the test set were not officially provided, we only used 40 cases to
randomly divide the training, validation and test sets of our experiments. The training set includes 32
cases (841 images), the validation set has 3 cases (48 images), and the remaining 6 cases are used as
the test set (160 images). The image size is uniformly 320×320.

**Comment by leiwb-coding** [comment]
The same Problem, and  i dont know the model whether handle 3D Spleen  data, or other  method  used extact 2d medical image  from Spleen

**Comment by wxshCoding** [comment]
I've encountered a similar issue. Can the Spleen 3D dataset mentioned in the paper be processed by the model, or is the Spleen dataset processed into a 2D data format?
